tests/generic_config_updater/test_dhcp_relay.py

```diff
@@ -0,0 +1,450 @@
+    |           |                  |           |                |             | 192.0.109.4           |
+    +-----------+------------------+-----------+----------------+-------------+-----------------------+
+    """
+    if init_dhcp_server_config != "default_setup":
+        pytest.skip("Unsupported init config")
+
+    dhcp_rm_json = [
+        {
+        "op": "remove",
+        "path": "/VLAN/Vlan" + str(vlan_intfs_list[0]) + "/dhcp_servers/3"
+        }]
+
+    tmpfile = generate_tmpfile(duthost)
+    logger.info("tmpfile {}".format(tmpfile))
+
+    output = apply_patch(duthost, json_data=dhcp_rm_json, dest_file=tmpfile)
+    expect_op_success_and_reset_check(duthost, output, 'dhcp_relay', DHCP_RELAY_THRESHOLD, DHCP_RELAY_INTERVAL, 0)
+    pytest_assert(duthost.is_service_fully_started('dhcp_relay'), "dhcp_relay service is not running")
+
+    unexpected_content_list = ["192.0." + str(vlan_intfs_list[0]) + ".4"]
+    expect_res_success_by_vlanid(duthost, vlan_intfs_list[0], [], unexpected_content_list)
+
+    delete_tmpfile(duthost, tmpfile)
+
+def test_dhcp_relay_tc6_add(duthost, setup_vlan, init_dhcp_server_config, vlan_intfs_list):
+    """Test normal add dhcp server on default setup
+
+    This VLAN detail should show below after test
+    +-----------+------------------+-----------+----------------+-------------+-----------------------+
+    |   VLAN ID | IP Address       | Ports     | Port Tagging   | Proxy ARP   | DHCP Helper Address   |
+    +===========+==================+===========+================+=============+=======================+
+    |       108 | 192.168.108.1/24 | Ethernet4 | tagged         | disabled    | 192.0.108.1           |
+    |           |                  |           |                |             | 192.0.108.2           |
+    |           |                  |           |                |             | 192.0.108.3           |
+    +-----------+------------------+-----------+----------------+-------------+-----------------------+
+    |       109 | 192.168.109.1/24 | Ethernet4 | tagged         | disabled    | 192.0.109.1           |
+    |           |                  |           |                |             | 192.0.109.2           |
+    |           |                  |           |                |             | 192.0.109.3           |
+    |           |                  |           |                |             | 192.0.109.4           |
+    |           |                  |           |                |             | 192.0.109.5           |
+    +-----------+------------------+-----------+----------------+-------------+-----------------------+
+    """
+    if init_dhcp_server_config != "default_setup":
+        pytest.skip("Unsupported init config")
+
+    dhcp_add_json = [
+        {
+            "op": "add",
+            "path": "/VLAN/Vlan" + str(vlan_intfs_list[1]) + "/dhcp_servers/4",
+            "value": "192.0." + str(vlan_intfs_list[1]) + ".5"
```

Comment:
>vlan_intfs_list[1] [](http://example.com/codeflow?start=36&length=18)

Is it possible that the value is too large and the IP is not valid? Check before use it this way?

Reply:
No. It is restricted by `utils_vlan_intfs_dict_add` which in range(0, 255)